data/json/npcs/factions.json

```diff
@@ -306,6 +306,35 @@
     ],
     "description": "An outpost started by the Free Merchants to provide a source of food and raw materials."
   },
+  {
+    "type": "faction",
+    "id": "isolated_artisans",
+    "name": "Isolated Artisans",
+    "likes_u": 10,
+    "respects_u": 0,
+    "known_by_u": false,
+    "size": 2,
+    "power": 5,
+    "food_supply": 4000,
+    "wealth": 100000,
+    "currency": "FMCNote",
+    "relations": {
+      "free_merchants": {
```

Comment:
So this is not how they treat the free merchants, it is how the free merchants treat them.  If I remember correctly.  So you are saying that the FMC would share goods freely without charging to these two individuals.  (I'm not sure all of these actually do anything yet but I want to make sure you know what this is saying)

Reply:
I did not! So thank you! They also would not freely share their goods